### ISSUE TITLE
Cannot use Logging capabilities

### ISSUE THREAD

**Issue by Melchizedech** [PaloAltoNetworks/terraform-provider-prismacloud#1]
## Describe the bug

It is not possible to benefit from the logging/debugging capabilities of the go SDK of PaloAlto while using this provider.

## Expected behavior

As outlined in the documentation, https://www.terraform.io/docs/providers/prismacloud/index.html#logging, when I add a mapping to enable logging, I have logging/debugging information output.

## Current behavior

The logging/debugging option is not taken into account.
Culprit: https://github.com/terraform-providers/terraform-provider-prismacloud/blob/58afd8f8a67ab2743c2da7440fc0f66b9269fd84/prismacloud/provider.go#L155

## Possible solution

Add the logging parameter when initializing the SDK client ?

## Steps to reproduce

1. using a provider configured this way : 
```
terraform {
  backend "gcs" {}
}

provider "prismacloud" {
  protocol = "https"
  url      = "api2.eu.prismacloud.io"
  logging = {
    "action" : true,
    "path" : true,
    "receive" : true,
    "send" : true
  }
}
```
2. using this resource
```
resource "prismacloud_account_group" "staging" {
  name        = "Staging clusters"
  description = "Group containing only staging custers"
}
```

3. with environment variables for username/password access
4. applying this configuration won't display any more information (gut is applied)

1. using the same provider 
2. using this resource

```
resource "prismacloud_integration" "slack" {
  name             = "Slack"
  integration_type = "slack"
  description      = "Alert channel"
  enabled          = true

  integration_config {
    url = "https://your.url"
  }
}
```
3. applying this plan will fail but no clue on what parameters are sent to the api (only the errors from the header are displayed)

## Context

I cannot create a cloud_account resource of type "GCP", I suspect a bug in the way it is handled.
I cannot create a Slack integration, I suspect a bug in the way it is handled.

I wanted to add logging to know what's sent/received as API calls via Postman are OK.
I can't use logging to pinpoint the issue easily.

## Your Environment

- Version used: 1.0.0
- go 1.13, terraform v0.12.28
- Linux XXX 5.7.9-arch1-1 PaloAltoNetworks/terraform-provider-prismacloud-orig#1 SMP PREEMPT Thu, 16 Jul 2020 19:34:49 +0000 x86_64 GNU/Linux

**Comment by ebeuerle** [comment]
This should be fixed as of v1.0.7 - please validate.

**Comment by ebeuerle** [comment]
I have validated that logging is working correctly.  Closing issue.
